modules/auxiliary/scanner/sap/sap_soap_rfc_eps_get_directory_listing.rb

```diff
@@ -36,7 +36,7 @@ def initialize
       },
       'References' =>
         [
-          [ 'URL', 'http://labs.mwrinfosecurity.com' ]
+          [ 'URL', 'https://labs.f-secure.com/' ]
```

Comment:
Should this be `'https://labs.f-secure.com/tools/sap-metasploit-modules/'` like the others?

I downloaded the zip, but didn't find this module in there (perhaps the file name has changed).

Reply:
I tried to do 1:1 replacements, investigating 2,600 broken URLs intelligently was entirely too time consuming.  I'm keen to leave this as is since that's what was originally there